src/main/java/org/cyclops/integrateddynamics/core/evaluate/variable/ValueTypeList.java

```diff
@@ -117,6 +117,10 @@ private ValueList(IValueTypeListProxy<T, V> value) {
             return values.length == 0 ? ValueTypes.LIST.getDefault() : ofList(values[0].getType(), ImmutableList.copyOf(values));
```

Comment:
I'm thinking of getting rid of this method in another PR. It's only used by testing code, and all uses can easily be replaced with the new overload I added.

Reply:
Sounds good!